### ISSUE TITLE
Clarification on SCE loss

### ISSUE THREAD

**Issue by lindsey98** [jeromerony/dml_cross_entropy#6]
Hi,
   Can I have a quick clarification on the SCE loss. Is the epsilon in your loss aim to avoid -inf problem for log? Thanks!

**Comment by jeromerony** [comment]
Hey,
The epsilon in the loss is to smooth the target. We get slightly better results with that.
Indeed, in theory, the optimal logits is not inf anymore. In practice though, the logits are never going to inf because of all the regularization though.

**Comment by lindsey98** [comment]
Ok thanks!
